src/scripts/dialog.js

```diff
@@ -107,7 +107,9 @@ const dialogConfig = (editor) => ({
     dialog.setValueOf('tab-basic', 'language', language);
 
     // set code and output
-    const widget = dialog.getModel(editor);
+    // getModel is not supported on older versions
+    // const widget = dialog.getModel(editor);
+    const widget = window.currentWidget;
```

Comment:
```suggestion
    const widget = window.currentThebelabWidget;
```
@moorepants I believe this naming convention is unique enough.